resources/lib/pigskin.py

```diff
@@ -228,6 +228,33 @@ def get_weeks_games(self, season, season_type, week):
             raise
 
         return sorted(games, key=lambda x: x['gameDateTimeUtc'])
+        
+    def get_team_games(self, season, team=None):
+        try:
+            url = self.config['modules']['ROUTES_DATA_PROVIDERS']['teams']
+            teams = self.make_request(url, 'get')
+            if team is None:
+                return teams
+            else:
+                # Looking for the Teamname
+                for conference in teams['modules']:
+                    if 'content' in teams['modules'][conference]:
+                        for teamname in teams['modules'][conference]['content']:
+                            if team == teamname['fullName']:
+                                team = teamname['seoname']
```

Comment:
Yeah, as soon as the team name is found the loop has served its purpose. Basically, you should add the ```break``` statement after line 244. :-)